tests/test_views.py

```diff
@@ -23,6 +23,33 @@ def test_study_create(client, user, study_area):
     assert study_area.name in response.content.decode('utf-8')
 
 
+@pytest.mark.django_db
+def test_study_list(client, user):
+    client.force_login(user)
+    url = reverse('studies-list')
+    response = client.get(url)
+
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_study_delete(client, user, study):
+    client.force_login(user)
+    url = reverse('studies-delete', kwargs={'pk': study.id})
+    response = client.get(url)
```

Comment:
I think this only tests that we can `GET` the delete page, but not that we can actually delete a study, right?

Reply:
Civic Exchange internet didn't push through my last commit extending this test 😬 It's in now testing both get and post!